packages/paste-core/components/toast/src/types.ts

```diff
@@ -29,17 +29,25 @@ export const ToastPropTypes = {
 export interface ToastPortalProps extends React.HTMLAttributes<HTMLDivElement> {
   children: NonNullable<React.ReactNode>;
   ref?: any;
+  // FIXME: Overrides token zIndex to fix bug with Console product.
+  __console_patch?: boolean;
```

Comment:
I hadn't thought of how this prop would work, and seeing it be a boolean, especially for a temp fix, makes so much sense.